kyuubi-metrics/src/test/scala/org/apache/kyuubi/metrics/MetricsSystemSuite.scala

```diff
@@ -41,11 +44,11 @@ class MetricsSystemSuite extends KyuubiFunSuite {
     }
   }
 
-  test("MetricsSystem") {
+  test("metrics - JsonReporter") {
     val reportPath = Utils.createTempDir()
     val conf = KyuubiConf()
       .set(MetricsConf.METRICS_ENABLED, true)
-      .set(MetricsConf.METRICS_REPORTERS, ReporterType.values.map(_.toString).toSeq)
```

Comment:
can we still have all non-PrometheusReporters?

Reply:
update to "metrics - PrometheusReporter" and "metrics - other reporters"